modules/core/src/main/scala/sangria/schema/Schema.scala

```diff
@@ -70,7 +70,7 @@ sealed trait AbstractType extends Type with Named {
       .flatMap(_.find(_.isInstanceOf(value)).asInstanceOf[Option[ObjectType[Ctx, _]]])
 }
 
-sealed trait MappedAbstractType[T] extends Type with AbstractType with OutputType[T] {
+sealed trait MappedAbstractType[T] extends Type with AbstractType {
   def contraMap(value: T): Any
 }
```

Comment:
I see that this change causes binary compatibility errors (for scala 2). I removed OutputType[T] as it would cause compiler conflicts in scala 3 (there is another place where an anonymous class is created with both OutputType[T] from MappedAbstractType and OutputType[Any] from UnionType). Removing it here did not cause any problems elsewhere, so I did not think much of it. Since the this is a sealed trait inside a file full of them, making a version specific versions of the file seems impractical and heavy on the maintenance work... Outside of breaking binary compatibility the only solution I can think of is something akin to https://github.com/akka/akka-http/pull/4113, where they have an sbt plugin that filters some implementations out of a single file depending on the scala version used

Reply:
You can also add some `mimaBinaryIssueFilters`.